Hotfix: Medbay Maintenance Upgrade (Delta/Kerberos)

ItsMeJaba opened the pull request:
Изначально предполагалось, что ПР #1094 с маркировкой [TEST MERGE] пойдет в тестовую ветку, но раз так, то хотя бы на тестовые перенести карту, а то она только на мейне и секонде.

# Fixes: 
* Исправлены декали в туалете, старой комнате мима и рядом с камерой для нагрева в токсинах;
* Вернул утерянную раковину генетикам;
* Исправлена зона юго-восточных солнечных панелей;
* Исправлены баррикады на входе в тоннели из раздевалки дорм;
* Исправлены названия фильтров в старой атмосии;
* Теперь из воздуха не запитываются: старое казино, старый офис детектива, старая телепортерная, северная ботаника и техи СБ.
* Приклеил кнопку затемнения в генетике на клей, а не сопли
* Теха вирусологии теперь защищают от радиации
* Исправлены тайлы пола, что скрывали трубы на себе
* Исправлены доступы для медиков в их же тех.тоннели
* Маска клувни заменена на обычную (нет смешным смертям🤡)

ItsMeJaba committed:
Горячие фиксики

Den109G commented:
:Starege:
`into testmerge`
Остальные сервера пока без фиксов побудут, видимо.

ItsMeJaba committed:
MicroFix

Выгнал генетика раундстартом из загона с мартышками

ItsMeJaba commented:
> :Starege: `into testmerge` Остальные сервера пока без фиксов побудут, видимо.

Вот бы я еще заметил, что прошлый ПР идет в мастер 2 месяца назад, хотя хотел мариновать на тестовой ветке👺

larentoun commented:
Ты и так замучал Азиза мерджем этой карты. Лучше лей патчи на мейны уже

Den109G commented:
Ничего-ничего. Будем тестить на основной ветке.

larentoun commented:
> Ничего-ничего. Будем тестить на основной ветке.

Я уже натестилась, в виде турфа пола, что сосет воздух, как турф космоса

ItsMeJaba committed:
WE NEED MORE FIXES

Поработал над проводами, доступами в теха меда для медиков, поправил мелкую настенную бню и магию

dikers93 commented:
> > Ничего-ничего. Будем тестить на основной ветке.
> 
> Я уже натестилась, в виде турфа пола, что сосет воздух, как турф космоса

ну критических недоработок нет, так что все в норме

Bizzonium commented:
Будет замержено только на master220 ветку, в остальных конфликты

ItsMeJaba commented:
> Будет замержено только на master220 ветку, в остальных конфликты

Решается откатом #1266 с последующим переносом пра #1094 и фикса на него #1382, либо форсом #1094 и добавлением #1382, целостность та же. Могу наплодить сущностей и сделать отдельный на это пр.
